backend/pkg/sqlmanager/postgres/postgres-manager.go

```diff
@@ -15,6 +15,8 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const PostgresDriver = "postgres"
```

Comment:
does this need to be exposed outside of this module?

Reply:
In the querybuilder we have a `GoquPostgresDriver`, should be we be using that on 878?

Reply:
Import cycle 😢 . moved it to the shared sqlmanager. So now we just have PostgresDriver and GoquPostgresDriver